src/Components/useFetch/useFetch.js

```diff
@@ -5,11 +5,16 @@ import { numberReq, url } from '../../Constants';
 const useFetch = () => {
   
     const [data, setData] = useState([]);
+    const [error, setError] = useState(null)
```

Comment:
The logic for useFetch and useFetchForm looks very similar, and both can be refactored to one file. We basically just send request to get data, using axios. Then we set some piece of state, and just return that value. I think we can return data and error, and in component pull the values from that data object.

Reply:
Still not resolved.

Reply:
I think I  resolved it, just forgot to delete useFetchForm component. I only use useFetch in my app for getting data.